src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -23,6 +23,8 @@
 TASK_NOT_SUPPORTED = 'Task is only supported for managed registries.'
 DEFAULT_TOKEN_TYPE = 'PAT'
 NULL_CONTEXT = '/dev/null'
+IDENTITY_LOCAL_ID = '[system]'
+IDENTITY_GLOBAL_REMOVE = '[all]'
```

Comment:
Same question, why not just `all`? Is this also case-sensitive?

Reply:
To keep with the precedent of `[system]` I used `[all]`. I will make this case insensitive